test/test_projection.py

```diff
@@ -76,6 +84,53 @@ def test_project(self):
 
         self.assertEqual(test_successful, True)
 
+    def test_targets(self):
```

Comment:
i backported this test into `develop` branch, but couldn't find an easy way to adapt `IProductionBenchmarkScopes`.
As such, we don't have a confirmation, that refactoring didn't break it.
I am thinking what to do with it

Reply:
Recall that a ProductionBenchmark is fundamentally a timeseries of year-over-year production growth numbers for a sector and a region: i.e.,
```
Steel, North America, 0.0, 0.01, 0.01, 0.015, 0.015, 0.02 etc
```
The production benchmark conveys no meaningful information about Scope; the Emissions and Emissions Intensity benchmarks DO have meaningful scope information.  However, we shape all the benchmarks similarly, so we add a meaningless Scope attribute to the production benchmark, which can be everywhere ignored.  We see it as "AnyScope" in the current implementation, but it used to be EScope.S1S2.  To adapt IProductionBenchmarkScopes, you can use the year-over-year timeseries and you can set the scope to be whatever your existing context expects (or can ignore).